pages/posts/portal-bridge-latest-guide.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to run Portal Bridge on history latest mode Windows/Linux guide
+date: 2024/2/15
+description: Explaining how to run a Portal Bridge in history latest mode.
+tag: portal bridge, Ethereum, archive
+author: Kolby ML (Moroz Liebl)
+---
+
+# What is a Portal Bridge?
+A Portal Bridge is a application which takes Ethereum mainnet data from a source and provides it to a Portal client to inject into Portal sub-networks.
+
+# What are we doing in this guide?
+We will be deploying a Portal Bridge using the free tier of infura. The bridge will be running the History latest mode, which gossips the latest Ethereum block data onto the Portal History network. The Portal History network is a Portal sub-network which stores canonical Ethereum mainnet headers, block bodies, and receipts.
```

Comment:
`The bridge will be running in "latest" mode for the History network, injecting the latest Ethereum block data into the History network layer of the Portal network.`